src/client/src/type-declarations/openfin.d.ts

```diff
@@ -6,4 +6,9 @@ declare namespace fin {
   interface ApplicationOptions {
     nonPersistent?: boolean
   }
+
+  interface OpenFinDesktop {
+    ExcelService: any
```

Comment:
Anyways to type this to something else other than any?

Reply:
added some extra types for this